src/main.py

```diff
@@ -60,6 +65,15 @@ def get_next_song(self) -> dict:
         if self.repeatqueue :
             if self.current_song_number == len(self.songs):
                 self.current_song_number = 0
+        if self.searchObject:
```

Comment:
Bhai ye error nahi dega aagr searchstr diya hi nahi hoga to ?

Reply:
ho gaya sahi ab